backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -166,5 +166,17 @@
             >Start
         </button>
     {% endif %}
+    
+    {% if card.content_type_nice == "project" or card.content_type_nice == "topic" and card.topic_needs_review %}
```

Comment:
We need to get rid of topic_needs_review. It's debt. Please don't build it into new logic